app/components/course/CourseDropDownButton.tsx

```diff
@@ -51,22 +51,30 @@ const CourseCardDropDown = ({
         {dropDownText}
       </div>
       {isOpen && (
-        <div className="absolute w-48 bg-white shadow-lg rounded-lg border border-solid border-gray-300 z-10">
+        <div className="absolute w-48 bg-white shadow-lg rounded-lg border border-solid border-gray-300 z-10 right-px">
           <ul>
             {dropDownText === "No Selection" && (
               <>
-                <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("Planned")}>Planned</li>
-                <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("In Progress")}>In Progress</li>
-                <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("Completed")}>Completed</li>
-              </>
-            )}
-            {dropDownText !== "No Selection" && (
-              <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("No Selection")}>Remove</li>
-            )}
-          </ul>
-        </div>
-      )}
-    </div>
+              <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("Planned")}>
+                Planned
+              </li>
+              <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("In Progress")}>
+                In Progress
+              </li>
+              <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("Completed")}>
+                Completed
+              </li>
+            </>
+          )}
+          {dropDownText !== "No Selection" && (
+            <li className="py-2 px-4 hover:bg-gray-100 cursor-pointer" onClick={() => handleOption("No Selection")}>
+              Remove
+            </li>
```

Comment:
Since the status is a static(doesn't change dynamically), why can't we just save a static constant somewhere and refer back to that variable and use Array.map instead?

Reply:
Is this referring to the sorting from the other comment? I'm a little confused.

Reply:
Is this referring to the sorting from the other comment? I'm a little confused.

Reply:
mostly the same. In that other comment, I mention having a value for each status. while this one is just have them as a list.